src/Util.cpp

```diff
@@ -60,67 +60,7 @@
 #include <mutex>
 #include <thread>
 
-#if __has_include(<pthread.h>) && !defined(Q_OS_WIN)
-// MacOS, Linux, etc
-#  include <pthread.h>
-static constexpr unsigned PLATFORM_STACK_MIN = PTHREAD_STACK_MIN;
```

Comment:
You could also just change this like to:

```c++
static const unsigned PLATFORM_STACK_MIN = PTHREAD_STACK_MIN;
```

And it would also fix #87 .. without needing to delete all this code...

Reply:
Follow your instinct. I'm a guest that have a preference for removing code, I have no knowledge if this code is/will be useful.

Reply:
TBH since it's not used and is rather nasty (and probably fragile, as we have seen) -- it's probably best to remove. It was used before but is no longer used.